.github/workflows/reusable_basic.yml

```diff
@@ -286,7 +310,10 @@ jobs:
         vcpkgJsonGlob: '**/vcpkg.json'
 
     - name: Install dependencies
-      run: vcpkg install --triplet x64-windows
+      run: |
+        vcpkg install --triplet x64-windows        
+        $env:Path = "${{env.VCPKG_PATH_BIN}};$env:Path"
```

Comment:
perhaps it's worth noting why it's needed (a comment here?)

Reply:
done